Readability.js

```diff
@@ -1854,6 +1854,12 @@ Readability.prototype = {
     // get site name
     metadata.siteName = jsonld.siteName || values["og:site_name"];
 
+    // get image thumbnail
+    metadata.image = values["og:image"] || values.image || values["twitter:image"];
```

Comment:
Do we also need to check `jsonld` for this? Would more work be required to extract this info from the JSON-LD bits?

What about the other prefixes, can images occur there?